pipettes/core/sensor_tasks.cpp

```diff
@@ -216,9 +228,14 @@ void sensor_tasks::QueueClient::send_pressure_sensor_queue_front(
     }
 }
 
-void sensor_tasks::QueueClient::send_tip_notification_queue(
+void sensor_tasks::QueueClient::send_tip_notification_queue_rear(
+    const sensors::tip_presence::TaskMessage& m) {
+    tip_notification_queue_rear->try_write(m);
```

Comment:
```suggestion
    if (tip_notification_queue_rear) tip_notification_queue_rear->try_write(m);
```
these (well, technically only the one that's optionally present, but really both) need to have their access to the `tip_notification_queue_{front,rear}` pointer guarded by a null check

Reply:
but style it better than the suggestion with a new line and braces and such